service/history/timerQueueAckMgr.go

```diff
@@ -64,8 +64,6 @@ type (
 		outstandingTasks map[TimerSequenceID]bool
 		// timer task ack level
 		ackLevel TimerSequenceID
-		// timer task read level
-		readLevel TimerSequenceID
```

Comment:
the read level is used by failover, as far as i remember

Reply:
added back